docs/utilities/jasix.mdx

```diff
@@ -0,0 +1,299 @@
+---
+title: JASIX
+---
+
+## Overview
+
+The JASIX index is aimed at providing TABIX like indexing capabilities for the Nirvana JSON output.
+
+## Creating the JASIX index
+
+The JASIX index (that comes in a .jsi) file is generated on-the-fly with Nirvana output. It can also be generated independently by running the Jasix command line utility on the JSON output file. Please note that the Jasix utility can only consume JSON files that follow the Nirvana JSON output format. The following code blocks demonstrate the help menu and index generating functionalities of Jasix.
+
+### Example
+
+```scss
+dotnet Jasix.dll -h
+USAGE: dotnet Jasix.dll -i in.json.gz [options]
+Indexes a Nirvana annotated JSON file
+
+OPTIONS:
+      --header, -t           print also the header lines
+      --only-header, -H      print only the header lines
+      --chromosomes, -l      list chromosome names
+      --index, -c            create index
+      --in, -i <VALUE>       input
+      --out, -o <VALUE>      compressed output file name (default:console)
+      --query, -q <VALUE>    query range
+      --help, -h             displays the help menu
+      --version, -v          displays the version
+```
+```scss
+dotnet Jasix.dll --index -i input.json.gz
+---------------------------------------------------------------------------
+Jasix                                               (c) 2017 Illumina, Inc.
+Stromberg, Roy, Lajugie, Jiang, Li, and Kang                          2.0.0
+---------------------------------------------------------------------------
+
+Ref Sequence chrM indexed in 00:00:00.2
+Ref Sequence chr1 indexed in 00:00:05.8
+Ref Sequence chr2 indexed in 00:00:06.0
+.
+.
+.
+Peak memory usage: 28.5 MB
+Time: 00:01:14.8
+```
+## Querying the index
+The Jasix query format is chr:start-end. If not provided, it assumes end=start. If only chr is provided, all entries for that chromosome will be provided.
+
+```scss
+dotnet Jasix.dll -i  input.json.gz chrM:5000-7000
+```
+
+```JSON
```

Comment:
I think it needs to be `json` instead of `JSON`. I didn't see any formatting in this section.